### ISSUE TITLE
Regions with labelledBy are being announced incorrectly

### ISSUE THREAD

**Issue by corinnaSchultz** [FreedomScientific/standards-support#102]
### Summary
http://mars.dequecloud.com/demo/RegionExample.htm#ex1
See this example page.
Press R to navigate through the page regions

### Expected result

> JAWS should have said "This week's poll region"

### Actual result

> Jaws says "heading level 3 this week's poll this week's poll region"

### Example

> http://mars.dequecloud.com/demo/RegionExample.htm#ex1

### Additional Information
The markup shows a div with a role="region", and a labelledBy referring to an h3 tag. I think the labelledBy is being confused and announcing the entire h3 tag instead of just the contents.

Also, when I use Ins+F3 to view a list of regions, the region labels say:
* This Week's Poll region  (no "heading")
* Content information

#### JAWS version and build number
JAWS 2018.1807.8 ILM
 
#### Operating System and version
Seen on Windows 7 professional SP1, and Windows 10

#### Browser and version:
Firefox 56.0, and Firefox 61.0.2 (on win7); 
Firefox 61 on Win10  (not sure of the minor version)

**Comment by jducrot** [comment]
@BenKeyFSI This is still happening in JAWS 2020.2004.66, see https://cdpn.io/TPG/debug/8bb3fad8b16b3fd9bed8265d50c7f804

**Comment by scottaohara** [comment]
is this actually an issue, or just how JAWS immediately navigates into landmark regions and begins announcing the first element within?

for instance, if the first element within the provided example were a paragraph with text "hello", JAWS would navigate to the region by use of the <kbd>R</kbd> key, announce "hello" as that is where the virtual cursor is placed, and then announce "This week's poll, region".

**Comment by jducrot** [comment]
@scottaohara I had the same question so I created the codepen included in my last comment. Have you checked it out?

**Comment by scottaohara** [comment]
just responding here to our offline convo, but yes.  checked both the original example and new pen out.

**Comment by JAWS-test** [comment]
> is this actually an issue, or just how JAWS immediately navigates into landmark regions and begins announcing the first element within?

I would say: It is a bug:
* The problem occurs in IE 11, Firefox and Chromium Edge. The problem does not occur in Chrome (surprisingly, Edge and Chrome are different - so far I don't know any other difference between the two browsers)
* The problem does not occur with all regions, e.g. not with the main element or role=main, but with the footer element and role=contentinfo
* Not only heading of the region is output, if it is referred to by aria-labelledby. Instead, the first child element is always output (even if it is text and even without aria-labelledby). 
* The problem also occurs if the region is explicitly labeled (e.g. with aria-label)

**Comment by carmacleod** [comment]
Please also see https://github.com/nvaccess/nvda/issues/7823#issuecomment-758073493 regarding redundant heading and region announcements. NVDA was reading the region label and then the (identical) heading label, but it isn't any more.
